src/components/CreateUser/CreateUser.tsx

```diff
@@ -1,24 +1,52 @@
 import React from "react";
 import { useState } from "react";
 import { FormEvent } from "react";
+import { NewUser } from "../../clients/apiClient";
+import { createNewUser } from "../../clients/apiClient";
+import { ChangeEvent } from "react";
 import './CreateUser.scss';
 
 export function CreateUser() {
     const [username, setUsername] = useState<string>("");
     const [password, setPassword] = useState<string>("");
     const [profileImageUrl, setProfileImageUrl] = useState<string>("");
     const [userBio, setUserBio] = useState<string>("");
+    const [userType, setUserType] = useState<string>("Member");
+    const [status, setStatus] = useState<string>("");
+
+    const userTypes = ["Member", "Admin"];
 
     function handleSubmit(event: FormEvent<HTMLFormElement>) {
-        event.preventDefault()
+        event.preventDefault();
+
+        const newUser: NewUser = {
+            username: username,
+            password: password,
+            userBio: userBio,
+            profileImageUrl: profileImageUrl,
+            userType: userType,
+        }
+
+        createNewUser(newUser)
+            .then(() => {
+                setStatus("Great! You have created a new user.")
+            })
+            .catch((e) => setStatus(e.message))
     }
 
+    const handleTypeChange = (event: ChangeEvent<HTMLSelectElement>) => {
+        setUserType(event.target.value);
+        console.log(username)
+        console.log(userType)
```

Comment:
Can we remove these logs now?

Reply:
Could we actually set the user type directly from the `onChange` attribute of the select? Instead of calling the `handleTypeChange` method?

Will add the suggestion further down